Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -939,6 +941,27 @@ defineSuite([
         });
     });
 
+    it('debugWireframe', function() {
+        // More precise test is in Cesium3DTileBatchTableSpec
+        return Cesium3DTilesTester.loadTileset(scene, tilesetUrl).then(function(tileset) {
+            viewRootOnly();
+            tileset.debugWireframe = true;
+            scene.renderForSpecs();
+            var commands = scene.frameState.commandList;
+            var i;
+            for (i = 0; i < commands.length; ++i) {
```

Comment:
Typically we factor out `commands.length` to its own variable for a slight performance gain. It won't matter for tests, but is a good habit for consistency purposes.